examples/slack-kit-example/README.md

```diff
@@ -1,38 +1,314 @@
+
+**Note: if the pop-up window does not close, double check that the `redirectUrl` matches your current environment.**
+
+From here, you can click `Next` to choose one or more Slack channels to associate with your object recipient.
+
+## Choosing Slack channels
+
+On this route, you can associate your Knock object recipient with one or more Slack channels using the `SlackChannelCombobox` component. It accepts a `slackChannelsRecipientObject` as a prop that specifies a `collection` and `objectId` for the recipient object:
+
+```
+"use client";
+
+import { SlackChannelCombobox } from "@knocklabs/react";
+
+export default function SlackChannelWrapper({
+  collection,
+  objectId,
+  className,
+}: {
+  collection: string;
+  objectId: string;
+  className: string;
+}) {
+  const slackChannelsRecipientObject = {
+    objectId: objectId,
+    collection: collection,
+  };
+  return (
+    <div className={className}>
+      <SlackChannelCombobox slackChannelsRecipientObject={slackChannelsRecipientObject} />
+    </div>
+  );
+}
+```
+
+As you select channels using the combobox input, the component will update the channel data for that object to store the Slack channel ids.
+
+![choose Slack channels](./images/choose-slack-channel.png)
+
+**Note: if you want to use a private Slack channel, you need to make sure that the Slack Bot has been invited to that channel.**
+
+At this point, Knock has all of the information it needs to send Slack messages to your selected channels. Click `Next` to examine the channel data for your object and tenant.
+
+## Examine channel data
+
+In this step, you're not required to take any action, but this page uses the Node SDK to pull channel data for your object recipient and tenant. Hopefully, this gives you an idea of how Knock is storing the data and these two abstractions interact.
+
+This data is pulled dynamically, so if you make different channel selections or disconnect and reconnect your Slack auth, the value for `access_token` and `connections.channelIds` will change as well.
+
+If you click `Next` you can test your workflow end-to-end.
+
+## Trigger your workflow
+
+In this step, you can see a code sample using the Node SDK of how you'd trigger your workflow. This is actually the code that runs in a server action when you submit the form on this page. Go ahead and submit a form to send a Slack message into your designated channels:
+
+![trigger a workflow](./images/trigger-workflow.png)
+
+## Wrapping up
+
+We know this is a long tutorial, but just imagine how long it would be if you had to do all of this without SlackKit components and APIs. If you need help implementing any of the code in this example app, consider joining our [Slack community](https://knock.app/join-slack). We've got over 500 other developers in there learning how to improve their notification experiences with Knock.
```

Comment:
I'm not sure we need to note this here but i'm not the marketing person so defer to y'all 🤓 

```suggestion
If you need help implementing any of the code in this example app, consider joining our [Slack community](https://knock.app/join-slack). We've got over 500 other developers in there learning how to improve their notification experiences with Knock.
```

Reply:
This might be my best line of copywriting, but I can save it for the blog post version